resources/assets/components/utilities/QuantityForm.js

```diff
@@ -0,0 +1,74 @@
+import gql from 'graphql-tag';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { useMutation } from '@apollo/react-hooks';
+
+export const QuantityFormFragment = gql`
+  fragment QuantityForm on Post {
+    id
+    quantity
+    impact
+  }
+`;
+
+const UPDATE_QUANTITY_MUTATION = gql`
+  mutation UpdateQuantityMutation($id: Int!, $quantity: Int!) {
+    updatePostQuantity(id: $id, quantity: $quantity) {
+      ...QuantityForm
+    }
+
+    ${QuantityFormFragment}
+  }
+`;
+
+const isInteger = string => /^\d+$/.test(string);
+
+const QuantityForm = ({ post }) => {
+  const [newQuantity, setNewQuantity] = useState(post.quantity);
+  const [updateQuantity, { loading }] = useMutation(UPDATE_QUANTITY_MUTATION, {
+    variables: { id: post.id },
+  });
+
+  const handleSubmit = () =>
+    updateQuantity({ variables: { quantity: Number(newQuantity) } });
+
+  return (
+    <>
```

Comment:
I don't think I've seen these empty tags used before, can you do this as a wrapper so you're still returning something valid?

Reply:
Yep! These are just a "short syntax" for [React fragments](https://reactjs.org/docs/fragments.html).

Reply:
Nice! That was my guess but I only remember seeing the full `<React.Fragment>` used before. I like this style better!